src/main/java/staffconnect/model/meeting/MeetDateTime.java

```diff
@@ -0,0 +1,66 @@
+package staffconnect.model.meeting;
+
+import static java.util.Objects.requireNonNull;
+import static staffconnect.commons.util.AppUtil.checkArgument;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a Meeting's starting time in the staff book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidMeetDateTime(String)}
+ */
+public class MeetDateTime {
+
+    public static final String MESSAGE_CONSTRAINTS = "DateTime should be of the correct format dd/mm/yyyy HH:mm";
+    public static final String VALIDATION_REGEX = "\\d{2}/\\d{2}/\\d{4} \\d{2}:\\d{2}";
+
+    private static final DateTimeFormatter PROCESS_FORMAT = DateTimeFormatter.ofPattern("dd/MM/yyyy HH:mm");
+
+    private final LocalDateTime value;
+
+
+    /**
+     * Constructs a {@code MeetDate}.
```

Comment:
I think you missed this out. Don't need to commit this suggestion here, you can do it in your local repository.
```suggestion
     * Constructs a {@code MeetDateTime}.
```